### PR TITLE
feat(postcard-service): Create postcard service and unit test

### PR DESCRIPTION
**Pull request by mpiercy827**
What: Create postcard service and tests.

Why: Want to be able to create new postcards from the dashboard through the API.

Details:
- Added bare bones Postcard service with create method (POST to /postcards).
- Includes unit test.

### CODE REVIEW

File: app/services/index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 'use strict';
 
-var Angular = require('angular');
-var Cookies = require('angular-cookies');
+var Angular  = require('angular');
+var Cookies  = require('angular-cookies');
```
Comment (robinjoseph08) [inline app/services/index.js]: Nit: while we align our equal signs, we tend to align them among each block of `require`s:

```js
var Angular = require('angular');
var Cookies = require('angular-cookies');

var API      = require('./api');
var Auth     = require('./auth');
var Postcard = require('./postcard');
var Session  = require('./session');
```

File: test/services/postcard.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+var Config  = require('../../config');
```
Comment (robinjoseph08) [inline test/services/postcard.spec.js]: Nit: we usually keep out module `require`s separate from our relative `require`s.

**Comment by robinjoseph08** [comment]
Some small stuff but LGTM!

We also usually avoid spaces in the category of the commit message:

```
feat(postcard-service): Create postcard service and unit test
```

File: test/services/postcard.spec.js (hunk 2 above)
Comment (elnaz) [inline test/services/postcard.spec.js]: [source](https://github.com/lob/wiki/blob/master/engineering/style-guides/node.md#group-internalexternal-dependencies)

**Comment by elnaz** [comment]
LGTM

**Pull request merged**